payments-core/src/main/java/com/stripe/android/payments/bankaccount/navigation/CollectBankAccountContract.kt

```diff
@@ -41,7 +41,7 @@ class CollectBankAccountContract :
     sealed class Args(
         open val publishableKey: String,
         open val stripeAccountId: String?,
-        open val clientSecret: String,
+        open val clientSecret: String?,
```

Comment:
Can we add a comment here explaining when this can be null?

Reply:
Also the contract might be a better place to map from internal to exposed results, and would also match what we do on connections - https://github.com/stripe/stripe-android/blob/73d7f24b46df6a971f65ddc79fb4e640291d8e55/financial-connections/src/main/java/com/stripe/android/financialconnections/launcher/FinancialConnectionsSheetForDataContract.kt#L37

Reply:
We need to return `CollectBankAccountResultInternal` from the `createInternal` callback. Unfortunately, this method cannot use the  the final "contracted" `CollectBankAccountResult`, since it requires a non-null `StripeIntent`.